classes/fullstack/project1/client/src/store/slices/auth.slice.ts

```diff
@@ -0,0 +1,21 @@
+import { PayloadAction, createSlice } from "@reduxjs/toolkit";
+import { Admin } from "../../types"
+
+type InitialState = Pick<Admin, 'username'>
+
+const initialState: InitialState = {
```

Comment:
refactor to include all attributes

Reply:
is it ok to include all attributes like this? role is not an optional attribute, so i decided to assign it to the lowest priority.

Reply:
set to role | null see comment in last review